File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.js
function solution(input) {
    const ARRAY = input.splice(1, input.length);
    let result = '';

    ARRAY.sort((a,b) => a - b).forEach(el => {
        result += el + '\n';
    })
    
    console.log(result);
}

const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,    
});

let input = [];
let list = [];

rl.on("line", function (line) {
   input.push(line)
}).on("close", function () {
    input.forEach((val) => {
       list.push(val.split(' ').map((el) => parseInt(el)));
    });
    solution(list);
    process.exit();
});

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.py
"""
1. 입력 -> 1줄: 수의 개수 / 2줄: 숫자 | 1000개 이하
2. 출력 -> 1줄씩 정렬하여 배출 -> sort -> join
"""
N = int(input())
arr = []

for _ in range(N): # 시간복잡도: O(N)
    arr.append(int(input()))

arr.sort()
result = "\n".join(map(str, arr))

print(result)